### PR TITLE
Feature/jututsu kaisen

### PR DESCRIPTION
**Pull request by Nikhil-Nirmal**
Added Jujutsu kaisen quiz page and rule page.
check https://master-minds-test-deploy.netlify.app/ for reference

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

File: quizs/Jk/jk-quiz.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
+@import url('https://fonts.googleapis.com/css2?family=M+PLUS+Code+Latin&family=Offside&display=swap');
+  
+body {  
+    margin: 0px;
+    padding: 0px;
+    color: var(--white);
+    background-color: black;
```
Comment (abhisekgupta001) [inline quizs/Jk/jk-quiz.css]: we can be consistent here and replace black with a variable for the same.

File: quizs/Jk/jk-quiz.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    font-size: 16px;
```
Comment (abhisekgupta001) [inline quizs/Jk/jk-quiz.css:8]: recommended to use rem or em for font sizes.

File: quizs/Jk/jk-quiz.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+}
+.logo{
+    padding: 2rem 0px 2rem 4rem;
+}
+:root {
+    --red: #9e0101;
+    --dark-red: #612626;
+    --white: white;
```
Comment (abhisekgupta001) [inline quizs/Jk/jk-quiz.css]: we can be consistent while using colors and use only hex or only RGB or only names(not recommended)

File: quizs/Jk/jk-quiz.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  
+body {  
+    margin: 0px;
+    padding: 0px;
+    color: var(--white);
+    background-color: black;
+    font-size: 16px;
+}
+.logo{
+    padding: 2rem 0px 2rem 4rem;
```
Comment (abhisekgupta001) [inline quizs/Jk/jk-quiz.css:11]: just 0 or 0rem would have worked fine here, rather than mixing up px and rem.

**Review by abhisekgupta001 — APPROVED** [review]
Suggested some changes related to mixing up of units, rest everything is good.

File: quizs/Jk/jk-quiz.css (hunk 3 above)
Comment (Nikhil-Nirmal) [reply quizs/Jk/jk-quiz.css]: okay sure will look into that 
and submit changes as soon as possible

**Pull request merged**